internal/app/singularity/remote_list.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package singularity
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/sylabs/singularity/internal/pkg/remote"
+)
+
+const listLine = "%s\t%s\n"
+
+// RemoteList prints information about remote configurations
+func RemoteList(configFile string) (err error) {
+	c := &remote.Config{}
+	file, err := os.OpenFile(configFile, os.O_RDONLY|os.O_CREATE, 0600)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("No Remote configurations")
```

Comment:
I think `error` massages are not suppose to have caps, so:

```suggestion
			return fmt.Errorf("no remote configurations")
```